### DISCUSSION TITLE
How to enable autocomplete for tailwind?

### DISCUSSION THREAD

**Discussion by okanji** [zed-industries/zed#6956] [Feature Requests]
I can't seem to use autocomplete for tailwind? Does it ship with the app? Or do I need to enable it/ download it?

**Comment by 2zqa** [comment]
Zed claims it's built-in, but it doesn't work for me out of the box either. https://zed.dev/docs/languages/tailwindcss. Maybe this can be a starting point for finding a fix.

I think the [language server needs to be enabled explicitly](https://zed.dev/docs/configuring-languages#choosing-language-servers), I haven't tried this yet. I'll report back if I get to this.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
